ffi/cffi-test.c

```diff
@@ -4,38 +4,29 @@
 
 #include "delta_kernel_ffi.h"
 
-void visit_callback(void* engine_context, KernelStringSlice path, int64_t size, const DvInfo *dv_info, const CStringMap *partition_values) {
+void visit_callback(void *engine_context, KernelStringSlice path, int64_t size, const DvInfo *dv_info, const CStringMap *partition_values)
+{
```

Comment:
My C IDE is reformatting this code. Do we have a formatting standard I can set it to follow? @nicklan

Reply:
I added a .clang-format file which will enforce your style